pyproject.toml

```diff
@@ -193,21 +179,22 @@ envlist = sqlalchemy{ 1.3, 1.4, 2.0, 2.0-sqlmodel }-{ asyncpg, noasyncpg }-{aws_
 
 [testenv]
 passenv = CI
-deps =
-    sqlalchemy1.3: sqlalchemy<1.4
-    sqlalchemy1.4: sqlalchemy>=1.4,<2
-    sqlalchemy2.0: sqlalchemy>=2
-    sqlalchemy2.0-sqlmodel: sqlalchemy>=2
-    sqlalchemy2.0-sqlmodel: sqlmodel
-    asyncpg: asyncpg
-    aws_rds_iam: boto3
-    pydantic1: pydantic<2
-    pydantic2: pydantic>=2
-
-extras =
-    tests
-
-commands = pytest -vv --cov={envsitepackagesdir}/fastapi_sqla --cov-report xml --cov-report html --junitxml=test-reports/pytest/junit.xml
+skip_install = true
+allowlist_externals = poetry
+commands_pre =
+    poetry install
+    sqlmodel: poetry install --extras "sqlmodel"
+    aws_rds_iam: poetry install --extras "aws_rds_iam"
+    asyncpg-noaws_rds_iam: poetry install --extras "asyncpg"
```

Comment:
we specify it here because else it would install asyncpg twice in `asyncpg-aws_rds_iam`?

Reply:
I assume your question is : why do we have `asyncpg-noaws_rds_iam` instead of just `asyncpg`? Good catch. It does make it more efficient, but at the cost of a more complex config, and is not consistent with how I've approached the other dependencies.

Reply:
addressed in https://github.com/dialoguemd/fastapi-sqla/pull/138/commits/05806c24041d7ed620b2851c8e0f59e731bc2647